nautobot_golden_config/models.py

```diff
@@ -835,7 +826,7 @@ def to_csv(self):
 
     def __str__(self):
         """Return a sane string representation of the instance."""
-        return self.platform.slug
+        return f"{self.platform.slug}"
```

Comment:
Why this? if need a string, would do `str(self.platform.slug)`

Reply:
Will change.